e2e/constants/selectors/formBuilder.ts

```diff
@@ -0,0 +1,41 @@
+export const FORMBUILDER_SELECTORS = {
```

Comment:
```suggestion
export const FORM_BUILDER_SELECTORS = {
```

"Form builder" contains two words. So it should be separated by an `_`.